components/nexus-bootstrap/src/main/java/org/sonatype/nexus/bootstrap/osgi/LauncherActivator.java

```diff
@@ -43,18 +48,34 @@ public void start(BundleContext bundleContext) throws Exception {
     final String args = bundleContext.getProperty("nexus-args");
 
     MDC.put("userId", SYSTEM_USERID);
-    server = new Launcher(basePath, null, null, args.split(","));
-    server.startAsync();
+    launcher = new Launcher(basePath, null, null, args.split(","));
+    launcher.startAsync();
+
+    connectorConfigurationServiceListener = new ConnectorConfigurationServiceListener(
+        bundleContext,
+        launcher.getServer()
+    );
+    final String connectorFilter = "(objectclass=" + ConnectorConfiguration.class.getName() + ")";
+    bundleContext.addServiceListener(connectorConfigurationServiceListener, connectorFilter);
+
+    // activate so far registered ones
+    ServiceReference<?> references[] = bundleContext.getServiceReferences((String) null, connectorFilter);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move the array designator from the variable to the type. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1197)